### ISSUE TITLE
Failed to add new tailscale node as lock signer

### ISSUE THREAD

**Issue by linux4life798** [tailscale/tailscale#12779]
### What is the issue?

I reinstalled Raspbian on a Raspberry Pi using the same hostname as before. It previously had tailscale configured and was added as a lock signing node. I just finished installing tailscale on the clean raspberry pi, I signed the raspberry pi's nodekey using my one remaining lock signing node, then I tried to add the raspberry pi as a second signing node, which failed as you can see bellow:

```bash
craig@somehost:~$ tailscale lock add tlpub:vaild-key-grip

error: 500 Internal Server Error: network-lock modify failed: modify network-lock keys: cannot add key {25519 1 [vaild-key-grip-bytes-in-in-decimal] map[]}: already exists
```

---

The interesting thing is that the raspberry pi shows `tlpub:0000000000000000000000000000000000000000000000000000000000000000` as the WrappingPubKey, when I run `tailscale lock status`.

### OS

Linux

### OS version

Debian trixie

### Tailscale version

1.68.2

### Bug report

* Machine attempting to add new signing node:

  `BUG-44dc906d32d027cd4bb72f88cec83b499bb2205d6b1bfa7513c4cab463757c8e-20240711050234Z-9c2819aa729c3750`
* On the newly installed raspberry pi

  `BUG-f332da471e4ee7176c39bdb7e3eb9042a94a4a978d7d23a6ab935d5c498f2b8d-20240711050355Z-be46a2984ea0b449`

**Comment by linux4life798** [comment]
False alarm. I thought the `KeyID:` field of `tailscale lock status` was the identifier you needed to use when running `tailscale lock add <public-key>`.

**Comment by linux4life798** [comment]
But, when I added a new node to the network (without pre-signing), it didn't show that the new node was waiting to be signed, when I ran `tailscale lock status` on the new raspberry pi node.

I ran bugreport on the raspberry pi node that failed to show the locked out node.
BUG-f332da471e4ee7176c39bdb7e3eb9042a94a4a978d7d23a6ab935d5c498f2b8d-20240711055408Z-5d9684c02537d775


I could, however, see the notification on the original signing node when I ran `tailscale lock status`.
